isolate_example/lib/page_two.dart

```diff
@@ -110,6 +110,7 @@ class InfiniteProcessIsolateController extends ChangeNotifier {
   Future<void> start() async {
     if (_running == false && _paused == false) {
       await createIsolate();
+      if (newIsolate == null) return;
```

Comment:
Did this solve the exception you were seeing in the logs, or does it still exist?